test/mocha/30-credential-status.js

```diff
@@ -106,13 +111,84 @@ function _startServer({app}) {
           type: 'StatusList2021Entry',
           statusPurpose: 'revocation',
           statusListIndex: '67342',
-          statusListCredential: slCredential.id
+          statusListCredential: slCredentialRevocation.id
         },
-        issuer: slCredential.issuer,
+        issuer: slCredentialRevocation.issuer,
+      };
+
+      // Status List 2021 Credential with statusPurpose `revocation`
+      slCredentialSuspension = {
+        '@context': [
+          'https://www.w3.org/2018/credentials/v1',
+          VC_SL_CONTEXT_URL
+        ],
+        id: `${BASE_URL}/status/5d3e7a97-1121-11ec-9b38-10bf48838a41`,
+        issuer: 'did:key:z6Mktpn6cXks1PBKLMgZH2VaahvCtBMF6K8eCa7HzrnuYLZv',
+        issuanceDate: '2022-01-10T04:24:12.164Z',
+        type: ['VerifiableCredential', 'StatusList2021Credential'],
+        credentialSubject: {
+          id: `${BASE_URL}/status/5d3e7a97-1121-11ec-9b38-10bf48838a41#list`,
+          type: 'StatusList2021',
+          statusPurpose: 'suspension',
+          encodedList: encodedList100k
+        }
+      };
+
+      // Unsigned 2021 Credential with "credentialStatus.statusPurpose"
+      // `suspension`
+      unsignedCredentialSl2021TypeSuspension = {
+        '@context': [
+          'https://www.w3.org/2018/credentials/v1',
+          VC_SL_CONTEXT_URL,
+          'https://w3id.org/security/suites/ed25519-2020/v1'
+        ],
+        id: 'urn:uuid:a0418a78-7924-11ea-8a23-10bf48838a41',
+        type: ['VerifiableCredential', 'example:TestCredential'],
+        credentialSubject: {
+          id: 'urn:uuid:4886029a-7925-11ea-9274-10bf48838a41',
+          'example:test': 'foo'
+        },
+        credentialStatus: {
+          id: `${BASE_URL}/status/5d3e7a97-1121-11ec-9b38-10bf48838a41#67342`,
+          type: 'StatusList2021Entry',
+          statusPurpose: 'suspension',
+          statusListIndex: '67342',
+          // intentionally point the statusListCredential to a sl credential
+          // with status purpose revocation.
+          statusListCredential: slCredentialSuspension.id
```

Comment:
Comment doesn't match what's being done here.

Reply:
Thanks for catching that, I've removed the comment [60cba23](https://github.com/digitalbazaar/bedrock-vc-verifier/pull/32/commits/60cba23cd5cd8298ab70021b00f4e0d08267a2cf)